### DISCUSSION TITLE
Would be nice to be able to have multiple course plans saved to share with Advising. And to filter courses by goals.

### DISCUSSION THREAD

**Discussion by ekbl1** [qu8n/pennplanner#8] [Feature ideas]
Would be nice to be able to save multiple course plans to share with Advising, or just to try out different degree and certificate options.

Also, for a given course plan, would be nice to have the courses relevant to selected goal(s) (i.e. selected degree or certificate(s)) highlighted or filtered in the panel of all available courses. 

E.g. if you wanted to to MCIT plus the Computer Systems certificate, it'd be nice to have just courses possible to finish requirements for one or both of those (depending on if you've graduated MCIT already) highlighted or filtered in the left all courses panel.

**Comment by qu8n** [comment]
Thank you, Elizabeth. This and your other recent posts are all great suggestions. I'll keep them in mind for future updates

**Comment by ekbl1** [comment]
I really enjoy using your app and shared my plan with my Advisor, Jordan & we discussed. Much easier to use tool than anything else I’ve seen! Thanks for making this, and sharing it!Sent from my iPhoneOn Dec 17, 2023, at 4:28 AM, Quan Nguyen ***@***.***> wrote:﻿
Thank you, Elizabeth. This and your other recent posts are all great suggestions. I'll keep them in mind for future updates

**Comment by qu8n** [comment]
Thank you for your kind words, and so glad to hear that you're finding it helpful! (Jordan is my advisor as well, and he's great)
